### PR TITLE
Swagger updated with working model for clients

### PR DESCRIPTION
**Pull request by danielmarker**
New Swagger for the project - updated to work with Clients but will need to be merged into develop.

### CODE REVIEW

**Comment by Kirosoft** [comment]
:+1:

File: application/deed/deed-api.json [hunk 1]
```diff
@@ -86,8 +89,8 @@
     },
     "definitions": {
         "title_number": {
-            "pattern": "^([A-Z]{0,3}[1-9][0-9]{0,5}|[0-9]{1,6}[ZT])$",
-            "type": "string"
+            "type": "string",
```
Comment (endamccormack) [inline application/deed/deed-api.json]: We can probably remove these are they are not being used anymore

**Comment by endamccormack** [comment]
:+1:

**Comment by danielmarker** [comment]
Good team work!

**Pull request merged**